lab-michael/lib/bodyParser.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
You don't need this file any longer. We're using the body-parser module via NPM.